docs/3.0rc/deploy/infrastructure-examples/serverless.mdx

```diff
@@ -1,9 +1,9 @@
 ---
-title: Push work to serverless computing infrastructure
+title: Run flows on serverless infra
```

Comment:
```suggestion
title: Run flows on serverless infrastructure
```

I think our unwritten style guide has us spelling out infrastructure. We should make that explicit. :)

Reply:
If infrastructure makes the title wrap, consider "compute" instead

Reply:
```suggestion
title: Run flows on serverless compute
```